Marlin/src/lcd/menu/menu_main.cpp

```diff
@@ -148,32 +177,32 @@ void menu_main() {
       MENU_ITEM(gcode, MSG_SWITCH_PS_ON, PSTR("M80"));
   #endif
 
-  #if ENABLED(SDSUPPORT)
-
-    //
-    // Autostart
-    //
-    #if ENABLED(MENU_ADDAUTOSTART)
-      if (!busy)
-        MENU_ITEM(function, MSG_AUTOSTART, card.beginautostart);
-    #endif
-
-    if (card.isDetected()) {
-      if(!card.isFileOpen()) {
-      MENU_ITEM(submenu, MSG_CARD_MENU, menu_sdcard);
-      #if !PIN_EXISTS(SD_DETECT)
-        MENU_ITEM(gcode, MSG_CHANGE_SDCARD, PSTR("M21"));  // SD-card changed by user
-      #endif
-      }
-    }
-    else {
-      #if !PIN_EXISTS(SD_DETECT)
-        MENU_ITEM(gcode, MSG_INIT_SDCARD, PSTR("M21")); // Manually init SD-card
+  #if DISABLED(ENCODER_TYPE_BUTTONS)
+    #if ENABLED(SDSUPPORT)
+      //
+      // Autostart
+      //
+      #if ENABLED(MENU_ADDAUTOSTART)
+        if (!busy)
+          MENU_ITEM(function, MSG_AUTOSTART, card.beginautostart);
       #endif
-      MENU_ITEM(function, MSG_NO_CARD, NULL);
-    }
-  #endif // SDSUPPORT
 
+      if (card.isDetected()) {
+        if(!card.isFileOpen()) {
+        MENU_ITEM(submenu, MSG_CARD_MENU, menu_sdcard);
+        #if !PIN_EXISTS(SD_DETECT)
+          MENU_ITEM(gcode, MSG_CHANGE_SDCARD, PSTR("M21"));  // SD-card changed by user
+        #endif
```

Comment:
If you could fix the indentation and code formatting in spots like this it will help a lot by freeing up maintainers' time to work on solving problems. Please review the [Coding Standards](http://marlinfw.org/docs/development/coding_standards.html) documentation.